src/sparsezoo/v2/model_directory.py

```diff
@@ -535,6 +544,22 @@ def _download(
         else:
             raise NotImplementedError()
 
+    @staticmethod
+    def _sample_outputs_list_to_dict(
+        directories: List[NumpyDirectory],
+    ) -> Dict[str, NumpyDirectory]:
+        engine_to_numpydir_map = {}
+        for directory in directories:
+            engine_name = directory.name.split("_")[-1]
+            if engine_name not in ENGINES:
```

Comment:
make sure that `torch`, `tensorflow_v1`, and `keras` are all included as valid engines

Reply:
absolutely